src/java/org/apache/ivy/osgi/util/VersionRange.java

```diff
@@ -306,7 +306,7 @@ public boolean isClosedRange() {
         return startVersion.equals(endVersion);
     }
 
-    public boolean contains(String versionStr) throws ParseException {
+    public boolean contains(String versionStr) {
```

Comment:
Not sure about this one, either: it's a consequence of previous changes to Version which no longer throws exceptions.